examples/equilibrium_temperature_co2.jl

```diff
@@ -0,0 +1,30 @@
+using Klimakoffer
+using Printf
+
+NT = 48 # Number of time-steps per year
+
+mesh = Mesh()
+
+model = Model(mesh, NT)
+
+discretization = Discretization(mesh, model, NT)
+
+year_start = 1950
+year_end = 2020
+year_delta = year_end-year_start+1
+
+co2_concentration_yearly = zeros(Float64,year_delta)
+mean_temperature_yearly = zeros(Float64, year_delta)
+
+co2_concentration_start = 315.01
+co2_concentration_yearly_delta = 1.39
+
+
+for years = 1950:2020
```

Comment:
```suggestion
for years in 1950:2020
```
We prefer `in` since it reads more naturally